packages/block-editor/src/hooks/style.js

```diff
@@ -128,6 +128,10 @@ export function addSaveProps( props, blockType, attributes ) {
 
 	const { style } = attributes;
 	const filteredStyle = omitKeysNotToSerialize( style, {
+		[ BORDER_SUPPORT_KEY ]: getBlockSupport(
```

Comment:
```suggestion
		'border': getBlockSupport(
```

Note that the `style` block attribute has the following shape: 

```
{
  "style": {
    "border": {
      "radius": "<value>",
      "width": "<value>",
     // etc
    }
}
```

So we can't use BORDER_SUPPORT_KEY yet because it is still experimental: `__experimentalBorder`.

Reply:
Ahh right, good spot! Thank you!